ts/indicators/price-envelopes.src.ts

```diff
@@ -0,0 +1,339 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+
+        class PriceEnvelopesIndicator extends SMAIndicator {
+            public data: Array<PriceEnvelopesIndicatorPoint>;
+            public drawGraph(): void;
+            public init(): void;
+            public getValues(
+                series: Series,
+                params: PriceEnvelopesIndicatorOptionsParams
+            ): (boolean|IndicatorMultipleValuesObject);
+            public nameBase: string;
+            public options: PriceEnvelopesIndicatorOptions;
+            public parallelArrays: Array<string>;
+            public pointArrayMap: Array<string>;
+            public pointClass: typeof PriceEnvelopesIndicatorPoint;
+            public points: Array<PriceEnvelopesIndicatorPoint>;
+            public pointValKey: string;
+            public toYData(
+                point: PriceEnvelopesIndicatorPoint
+            ): [number, number, number];
+            public translate(): void;
+        }
+
+        interface PriceEnvelopesIndicatorGappedExtensionObject {
+            [index: string]: Dictionary<(number|undefined)>;
+        }
+
+        interface PriceEnvelopesIndicatorOptionsParams

```

Comment:
Has to be named `PriceEnvelopesIndicatorParamsOptions`.

Reply:
Fixed